tools/st2_deploy.sh

```diff
@@ -374,6 +395,16 @@ install_webui() {
   mkdir -p /opt/stackstorm/static/webui
   cp -R ${temp_dir}/* /opt/stackstorm/static/webui
 
+  # Replace config.js
+  echo -e "'use strict';
```

Comment:
That's fine for now, but if needed or wanted, we can also move this to the build step.